app/models/mixins/seeding_mixin.rb

```diff
@@ -0,0 +1,51 @@
+module SeedingMixin
+  extend ActiveSupport::Concern
+  # Seeding mixin can be used as long as fixture association
+  # are not nested but of the form: (for a hash_many relationship
+  #   :miq_policies:
+  #   - c5ceab70-da38-11e5-9cae-0242782a2608
+  #   - de0650f6-2412-4bea-b272-85ae7c9f61f3
+  module ClassMethods
+    def seed_model(model, options = {})
+      filter = options[:find_by] || -> (hash) { {:guid => hash[:guid]} }
+
+      load_fixtures(model.name.tableize).each do |hash|
+        association = options[:has_many].nil? ? nil : options[:has_many][0]
+        rec = seed_attributes!(model, hash.except(association), filter)
+        seed_relationships(rec, hash, options) if options[:has_many]
```

Comment:
I'd prefer you pass down the association and add_proc (or a better name for it) instead of an options hash.